core/include/mm/core_memprot.h

```diff
@@ -67,17 +67,22 @@ bool core_pbuf_is(uint32_t flags, paddr_t pbuf, size_t len);
 bool core_vbuf_is(uint32_t flags, const void *vbuf, size_t len);
 
 /*
- * Translate physical address to virtual address using specified mapping
+ * Translate physical address to virtual address using specified mapping.
+ * Also tries to find proper mapping which have counterpart translation
+ * for specified length of data starting from given physical address.
+ * Len parameter can be set to 1 if caller knows that requested (pa + len)
+ * doesn`t cross granule boundary.
```

Comment:
suggestions:
 ...for specified **byte** length of ....
 ...doesn`t cross **mapping** granule boundary.

Reply:
byte seems exceeding for me)